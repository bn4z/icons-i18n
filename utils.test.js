const utils = require('./utils');

test('json2String', () => {
    expect(utils.json2String({
        a: 1,
        b: {
            c: 2
        }
    })).toMatchSnapshot()
});

test('removeExtraSpaces', () => {
    expect(utils.removeExtraSpaces("  hello    world    ")).toBe("hello world")
});

test('removeWords', () => {
    expect(utils.removeWords("foo bar baz afoo bar baz", ["foo", "bar"])).toBe("baz afoo baz")
});