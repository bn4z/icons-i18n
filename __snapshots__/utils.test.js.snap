// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`json2String 1`] = `
"{
  \\"a\\": 1,
  \\"b\\": {
    \\"c\\": 2
  }
}"
`;
